### PR TITLE
Merge B matrix-related applications

### PR DESCRIPTION
**Pull request by benjaminmenetrier**
## Description

Companion PR of https://github.com/JCSDA-internal/saber/pull/573

### CODE REVIEW

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -42,7 +42,7 @@ ecbuild_bundle( PROJECT atlas GIT "https://github.com/ecmwf/atlas.git" TAG 0.31.
 # Core JEDI repositories
 # ----------------------
 ecbuild_bundle( PROJECT oops  GIT "https://github.com/jcsda/oops.git"  BRANCH develop UPDATE )
```
Comment (andreapiacentini) [inline bundle/CMakeLists.txt]: Why not feature/Bapps for oops ?  
Saber PR #573 depends on oops PR #2213 but the test names change and the Randomized Variance section should disappear from references ...  
Shouldn't AQ be ready for dealing with both saber and oops PR at once ?

**Review by andreapiacentini — CHANGES_REQUESTED** [review]

File: bundle/CMakeLists.txt (hunk 1 above)
Comment (benjaminmenetrier) [reply bundle/CMakeLists.txt]: You're correct, I forgot to change the oops branch.

Comment (andreapiacentini) [reply bundle/CMakeLists.txt]: I've moved the bundled oops and removed the Randomized Variance reference.  
I don't think we should mode to single obs with AQ: the current dirac tests are meaningful.  
On my side it is approved. Please, check if I missed something relevant in the oops upgrade

**Review by andreapiacentini — APPROVED** [review]

File: bundle/CMakeLists.txt (hunk 1 above)
Comment (benjaminmenetrier) [reply bundle/CMakeLists.txt]: Looks good to me.

**Comment by shlyaeva** [comment]
oops and saber PRs were merged, this PR can be merged now.

**Comment by benjaminmenetrier** [comment]
This PR can be merged now.

**Comment by andreapiacentini** [comment]
Thanks @shlyaeva and @benjaminmenetrier Life is easy for me !

**Pull request merged**